retrieval/discovery/dns.go

```diff
@@ -25,12 +25,18 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 
 	clientmodel "github.com/prometheus/client_golang/model"
-
 	"github.com/prometheus/prometheus/config"
-	"github.com/prometheus/prometheus/utility"
 )
 
-const resolvConf = "/etc/resolv.conf"
+const (
+	resolvConf = "/etc/resolv.conf"
+
+	DNSSourcePrefix = "dns"
```

Comment:
IMO we can still change it when needed. Capitalizing a name is a signal to the reader of the code that a name is used from somewhere else, which is misleading here (until it really is)...